select_target/select_target.gd

```diff
@@ -0,0 +1,25 @@
+extends CanvasLayer
+
+signal terminate(target)
+var player: Player
+
+
+func display(player: Player):
+	print("Choosing target")
+	self.player = player
+	
+	for target in State.players:
+		if target != player:
+			if target.sabotage_card:
+				target.sabotage_card = false
+			else: 
+				var button = preload("res://sabotage_field/target_button.tscn").instantiate()
+				$GridContainer.add_child(button)
```

Comment:
Nodes from scenes should be selected only once with `@onready` to improve maintainability.